ods_ci/tests/Tests/100__deploy/130__operators/150__dsp_operator/150__dsp_operator.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Documentation       Test Cases to verify Serverless installation
+
+Library             Collections
+Library             SeleniumLibrary
+Library             OpenShiftLibrary
+Resource            ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource            ../../../../Resources/OCP.resource
+Resource            ../../../../Resources/RHOSi.resource
+
+Suite Setup         Suite Setup
+Suite Teardown      Suite Teardown
+
+
+*** Test Cases ***
+Detect Pre-existing Install Of Argo Workflows And Block RHOAI Install
+    [Documentation]    Detect Pre-existing Install Of Argo Workflows And Block RHOAI Install
+    [Tags]                  Operator                ODS-2651                Tier1
+    ${return_code}          ${output}               Run And Return Rc And Output
+    ...                     oc apply -f ods_ci/tests/Resources/Files/argo/crd.workflows.yaml
+    Log To Console          ${output}
+    Should Be Equal As Integers
+    ...                     ${return_code}
+    ...                     0
+    ...                     msg=Error while applying the provided file
+    Open Installed Operators Page
+    Navigate to Installed Operators
+    ${is_operator_installed}                        Is Operator Installed                           ${OPERATOR_NAME}
```

Comment:
could you try avoiding OCP UI to check operators? it's been always flaky as method

Reply:
So Far it is working fine..

Reply:
Agree with Berto ..will suggest to go with cli approach ..it will cause problem between different openshift version

Reply:
ok changing